src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/config/ManageUsersApiExceptionHandler.kt

```diff
@@ -80,6 +91,24 @@ class HmppsManageUsersApiExceptionHandler {
       )
   }
 
+  @ExceptionHandler(HttpMessageNotReadableException::class)
+  fun handleValidationAnyException(e: HttpMessageNotReadableException): ResponseEntity<ErrorResponse?>? {
+    log.info("Validation exception: {}", e.message)
+    return ResponseEntity
+      .status(BAD_REQUEST)
+      .contentType(APPLICATION_JSON)
+      .body(ErrorResponse(status = BAD_REQUEST, userMessage = e.message, developerMessage = e.message))
+  }
+
+  @ExceptionHandler(MethodArgumentTypeMismatchException::class)
+  fun handleValidationAnyException(e: MethodArgumentTypeMismatchException): ResponseEntity<ErrorResponse?>? {
```

Comment:
We usually match the function name to the exception thrown - although looks like existing line 113 also needs updating